storage/innobase/include/fil0crypt.h

```diff
@@ -439,16 +397,10 @@ fil_parse_write_crypt_data(
 @param[in]		page_size	Page size
 @param[in,out]		dst_frame	Output buffer
 @return encrypted buffer or NULL */
-byte*
-fil_encrypt_buf(
-	fil_space_crypt_t*	crypt_data,
-	ulint			space,
-	ulint			offset,
-	lsn_t			lsn,
-	const byte*		src_frame,
-	const page_size_t&	page_size,
-	byte*			dst_frame)
-	MY_ATTRIBUTE((warn_unused_result));
+byte *fil_encrypt_buf(fil_space_crypt_t *crypt_data, ulint space, ulint offset,
```

Comment:
Doxygen comment mismatch. All parameter should be documented.

Reply:
This function was not implemented in our port, removed the declaration.